gobblin-runtime/src/main/java/gobblin/runtime/api/JobExecutionDriver.java

```diff
@@ -42,9 +45,9 @@
   //void stopForcefully();
 
   /** {@inheritDoc} */
-  @Override JobExecutionResult get() throws InterruptedException;
+  @Override JobExecutionResult get() throws InterruptedException, ExecutionException;
```

Comment:
Is adding ExecutionException necessary? The idea is to make the execution exceptions accessible through the errorCause in JobExecutionResult.